trl/trainer/ppo_trainer.py

```diff
@@ -41,6 +42,42 @@
 from trl.trainer.utils import AdaptiveKLController, FixedKLController
 
 
+MODEL_CARD_TEMPLATE = """---
+license: apache-2.0
+tags:
+- trl
+- transformers
+- reinforcement-learning
+- human-feedback
+---
+
+# {model_name}
+
+This is a [TRL model](https://github.com/lvwerra/trl) that can be used for FILL ME OUT.
+
+## Usage
+
+To use this model for inference, first install the TRL library:
+
+```bash
+python -m pip install trl
+```
+
+You can then run inference as follows:
+
+```python
+from transformers import AutoTokenizer
+from trl import AutoModelForCausalLMWithValueHead
+
+tokenizer = AutoTokenizer.from_pretrained({model_name})
+model = AutoModelForCausalLMWithValueHead.from_pretrained({model_name})
```

Comment:
Currently this will load a new value head as the `save_pretrained` method of `AutoModelForCausalLMWithValueHead` does not save the value head. I think we should save it. I don't know if @younesbelkada wants to add the necessary changes to this or a follow-up PR.

Reply:
Actually, to use the model you don't need TRL and you could just load it with transformers `AutoModelForCausalLM`. Installing TRL and the current example could be a second example if you want to continue training or get the value of the value head.

Reply:
Yes I can take care of that in a follow-up PR

Reply:
it has been taken care in https://github.com/lvwerra/trl/pull/86 !